src/torchmetrics/metric.py

```diff
@@ -142,8 +142,23 @@ def __init__(
     @property
     def _update_called(self) -> bool:
         # Needed for lightning integration
+        rank_zero_warn(
+            "Property `._update_called` have been made public in v0.12 and `.update_called` can therefore"
+            " be used. Will be removed in v0.13"
+        )
```

Comment:
```suggestion
```
I do not think this is needed as user shall not be using protected attributes...

Reply:
said so I would just rename it... :otter:

Reply:
alright I just rename :)